src/sqlfluff/dialects/dialect_redshift.py

```diff
@@ -283,6 +283,81 @@ class CreateTableAsStatementSegment(BaseSegment):
     )
 
 
+@redshift_dialect.segment()
+class CreateExternalTableStatementSegment(BaseSegment):
+    """A `CREATE EXTERNAL TABLE` statement.
+
+    As specified in https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_EXTERNAL_TABLE.html
+    """
+
+    type = "create_external_table_statement"
+
+    match_grammar = Sequence(
+        "CREATE",
+        "EXTERNAL",
+        "TABLE",
+        Ref("TableReferenceSegment"),
+        Bracketed(
+            OneOf(
+                # Columns and comment syntax:
+                Delimited(
+                    Sequence(
+                        Ref("ColumnReferenceSegment"),
+                        Ref("DatatypeSegment"),
+                    ),
+                ),
+            )
+        ),
+        OneOf(Ref("PartitionedBySegment"), optional=True),
+        "STORED",
+        "AS",
+        OneOf(
+            "PARQUET",
+            "RCFILE",
+            "SEQUENCEFILE",
+            "TEXTFILE",
+            "ORC",
+            "AVRO",
+            Ref("QuotedLiteralSegment"),
+        ),
+        "LOCATION",
+        Ref("QuotedLiteralSegment"),
+    )
+
+
+@redshift_dialect.segment()
+class CreateExternalTableAsStatementSegment(BaseSegment):
+    """A `CREATE EXTERNAL TABLE AS` statement.
+
+    As specified in https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_EXTERNAL_TABLE.html
+    """
+
+    type = "create_external_table_statement"
+
+    match_grammar = Sequence(
+        "CREATE",
+        "EXTERNAL",
+        "TABLE",
+        Ref("TableReferenceSegment"),
+        OneOf(Ref("PartitionedBySegment"), optional=True),
+        "STORED",
+        "AS",
+        OneOf(
+            "PARQUET",
+            "RCFILE",
+            "SEQUENCEFILE",
+            "TEXTFILE",
+            "ORC",
+            "AVRO",
+            Ref("QuotedLiteralSegment"),
+        ),
```

Comment:
```suggestion
        OneOf(
            "PARQUET",
            "TEXTFILE",
        ),
```
From the docs:
`The CREATE EXTERNAL TABLE AS command only supports two file formats, TEXTFILE and PARQUET.`

Reply:
good catch, thanks!